README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
it could be a config option on 2.x line, however I like the current setup :)
(since most of my projects are not yet on PHP 7 / `symfony/polyfill`)